modules/claims_api/app/workers/claims_api/ews_updater.rb

```diff
@@ -18,27 +18,35 @@ def perform(ews_id)
       else
         bgs_claim[:bnft_claim_dto][:filed5103_waiver_ind] = FILE_5103
 
-        response = bgs_service(ews).benefit_claims.update_bnft_claim(claim: bgs_claim)
-        if response[:bnft_claim_dto].nil?
-          ews.status = ClaimsApi::EvidenceWaiverSubmission::ERRORED
-          ews.bgs_error_message = "BGS Error: update_record failed with code #{response[:return_code]}"
-          ClaimsApi::Logger.log(ews_id: ews.id, claim_id: ews.claim_id,
-                                detail: 'Waiver update Failed', error: response[:return_code])
-        else
-          ews.status = ClaimsApi::EvidenceWaiverSubmission::UPDATED
-          # Clear out the error message if there were previous failures
-          ews.bgs_error_message = nil if ews.bgs_error_message.present?
-          ClaimsApi::Logger.log(ews_id: ews.id, claim_id: ews.claim_id, detail: 'Waiver Success')
-        end
+        get_response(ews, bgs_claim)
```

Comment:
I feel like `get_response` is bland. I think we just rename this but other than that I'm fine with the change. 

maybe `update_bgs_claim` or something to be a little more descriptive.